crates/bevy_render/src/view/mod.rs

```diff
@@ -116,12 +116,26 @@ impl ExtractedView {
     }
 }
 
+/// Configures basic color grading parameters to adjust the image appearance. Grading is applied just before/after tonemapping for a given [`Camera`](crate::camera::Camera) entity.
 #[derive(Component, Reflect, Debug, Copy, Clone, ShaderType)]
 #[reflect(Component)]
 pub struct ColorGrading {
+    /// Exposure value (EV) offset, measured in stops.
     pub exposure: f32,
+
+    /// Non-linear luminance adjustment applied before tonemapping. y = pow(x, gamma)
     pub gamma: f32,
+
+    /// Saturation adjustment applied before tonemapping.
+    /// Values below 1.0 desaturate, with a value of 0.0 resulting in a grayscale image
+    /// with luminance defined by ITU-R BT.709.
+    /// Values above 1.0 increase saturation.
     pub pre_saturation: f32,
+
+    /// Saturation adjustment applied after tonemapping.
+    /// Values below 1.0 desaturate, with a value of 0.0 resulting in a grayscale image
+    /// with luminance defined by ITU-R BT.709
+    /// Values above 1.0 increase saturation.
```

Comment:
This is all great. The missing piece I think is information about when/how to use them. Maybe that's a topic for elsewhere though.